include/seqan3/alphabet/composition/union_composition.hpp

```diff
@@ -207,24 +207,15 @@ class union_composition
     //!\brief The type of the alphabet when represented as a number (e.g. via \link to_rank \endlink).
     using rank_type = detail::min_viable_uint_t<value_size>;
 
-    /*!\name Default constructors
+    /*!\name Constructors, destructor and assignment
      * \{
      */
     constexpr union_composition() = default;
     constexpr union_composition(union_composition const &) = default;
     constexpr union_composition(union_composition &&) = default;
-    //!\}
-
-    /*!\name Default assignment operators
-     * \{
-     */
     constexpr union_composition & operator= (union_composition const &) = default;
     constexpr union_composition & operator= (union_composition &&) = default;
```

Comment:
down below you removed the spaces

to remove, or not to remove? ...

Reply:
to remove :performing_arts: 
(otherwise I cannot not \copydoc the function declaration in doxygen)